README.md

```diff
@@ -402,7 +402,7 @@ Output:
         thandoodstrenterprourete: [ '_reader', '_writer' ],
         nodejs: [ '_reader', '_writer', '_admin', '_replicator' ] } }
 
-See the Cloudant API for full details](https://docs.cloudant.com/api.html#authorization)
+See the [Authorization Documentation](https://console.bluemix.net/docs/services/Cloudant/api/authorization.html#authorization) for further details.
```

Comment:
Since there are reference links in this document we may as well use them, if they'd been used first time around it would have been easier for this PR.
>See the [Authorization] documentation for further details